app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -205,16 +210,15 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
 
   const { markets } = useGraphMarketsFromQuestion(loadedQuestionId || '')
 
-  const [currentToken, setCurrentToken] = useState({ exists: false, token: '', marketAddress: '' })
+  const [currentToken, setCurrentToken] = useState({ tokenExists: false, symbol: '', marketAddress: '' })
   useEffect(() => {
-    const rinkebyDai = '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea'
-    const mainnetDai = '0x6b175474e89094c44da98b954eedeac495271d0f'
     const array = markets.find(
-      ({ collateralToken }) => collateralToken === rinkebyDai || collateralToken === mainnetDai,
+      ({ collateralToken }) =>
+        collateralToken === DEFAULT_TOKEN_ADDRESS_RINKEBY || collateralToken === DEFAULT_TOKEN_ADDRESS,
     )
     setCurrentToken({
-      exists: array ? true : false,
-      token: 'DAI',
+      tokenExists: array ? true : false,
```

Comment:
I actually meant to change the `array` variable to something else, particularly since it shouldn't even return an array if I'm understanding correctly